sdk/containerregistry/perf-tests/container-registry/test/listRepositories.spec.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { PerfStressOptionDictionary } from "@azure/test-utils-perfstress";
+import { MetricsAdvisorTest } from "./client.spec";
+type MetricsAdvisorTestOptions = Record<string, unknown>;
+
+export class RepositoryListTest extends MetricsAdvisorTest<MetricsAdvisorTestOptions> {
+  public options: PerfStressOptionDictionary<MetricsAdvisorTestOptions> = {};
+  constructor() {
+    super();
+  }
+
+  async runAsync(): Promise<void> {
+    const listIterator = this.client.listRepositoryNames();
```

Comment:
Not creating resources?

Reply:
The library itself doesn't support this and relies on other ways to import or push images (az cli/powershell/management client libs).